components/trello/actions/update-card/update-card.mjs

```diff
@@ -1,102 +1,147 @@
-// legacy_hash_id: a_2wimVb
-import { axios } from "@pipedream/platform";
+import common from "../common.js";
+import pickBy from "lodash/pickBy.js";
+import pick from "lodash/pick.js";
 
 export default {
+  ...common,
   key: "trello-update-card",
   name: "Update Card",
-  description: "Updates the specified card.",
-  version: "0.1.1",
+  description: "Updates a card. [See the docs here](https://developer.atlassian.com/cloud/trello/rest/api-group-cards/#api-cards-id-put)",
+  version: "0.1.2",
   type: "action",
   props: {
-    trello: {
-      type: "app",
-      app: "trello",
-    },
-    id: {
+    ...common.props,
+    board: {
```

Comment:
Change this prop's name to `idBoard` (expected in `opts`, below)?

```js
  async run({ $ }) {
    const opts = pickBy(pick(this, [
      // ...
      "idBoard",
```

```suggestion
    idBoard: {
```

And update the inputValues functions for the props in this file. E.g.,
```js
    idCard: {
      propDefinition: [
        common.props.trello,
        "cards",
        (c) => ({
          board: c.idBoard,
        }),
      ],
```

---

Alternatively, add `idBoard` to `opts` using `this.board`:
```js
    const opts = pickBy(pick(this, [
      "name",
      "desc",
      "closed",
      "idMembers",
      "idAttachmentCover",
      "idList",
      "idLabels",
      "pos",
      "due",
      "dueComplete",
      "subscribed",
      "address",
      "locationName",
      "coordinates",
    ]));
    opts.idBoard = this.board;
```
with diff:
```diff
-     "idBoard",
      "pos",
      "due",
      "dueComplete",
      "subscribed",
      "address",
      "locationName",
      "coordinates",
    ]));
+   opts.idBoard = this.board;
```